python_utils.py

```diff
@@ -431,3 +431,15 @@ def yaml_from_dict(dictionary, width=80):
     """
     dictionary = _recursively_convert_to_str(dictionary)
     return yaml.safe_dump(dictionary, default_flow_style=False, width=width)
+
+def dict_from_yaml(yaml_file):
+    """Gets the dict representation of a yaml file.
+
+    Args:
+        yaml_file: yaml. Yaml file for conversion into dict.
+
+    Returns:
+        dict. Converted dict of the passed yaml file.
+    """
+    dictionary = yaml.safe_load(yaml_file)
+    return dictionary
```

Comment:
Add a newline at the end of the file. We expect this to fail during pre-push lint tests, can you please check how it passes the pre-push tests?

Reply:
While commiting or pushing, my tests do not trigger.

Reply:
Check whether you have the pre-push symlink in the .git/hooks/ folder?
![image](https://user-images.githubusercontent.com/16653571/71716451-d1bc9780-2e3a-11ea-8160-2fb5f56f4c46.png)

Reply:
Ah! It was present, i removed that and installed the hooks again. Got it working now.